express-notes-api/server.js

```diff
@@ -0,0 +1,155 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+const app = express();
+app.use(express.json());
+
+// errors array
+const errorArr = [
```

Comment:
This is a clever way to isolate all the error messages and avoid repeating them. I think it would be more clear accessing them if this were instead an object with keys equal to some string that describes the error and values as you have here in the array.

This way, instead of referencing the "Cannot find" message with `errorArr[1]` you would use something like `errorArr.notFound`.